recipes/default.rb

```diff
@@ -51,3 +51,10 @@
   user node['timesync']['user']
   action :start_or_graceful_reload
 end
+
+directory '/opt/timesync/source' do
```

Comment:
https://github.com/osuosl-cookbooks/nodejs-webapp/blob/master/resources/default.rb#L32

So it looks like this is creating the user, but not using the user to clone the git repo